client/instrumentation/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "substrate-instrumentation"
```

Comment:
I wonder if we can come up with a more specific name for that? Instrumentation sounds a bit too geeneral and for me implies a totally different thing.

How about "substrate-timings"?

Reply:
I wasn't totally sure about the name and I'm happy to make this change if people want it. The reason I went with instrumentation is because it's possible we might want to use it for other forms of instrumentation in addition to, or other than, timing. There's potential to selectively capture a lot of information using this tool - I'm not sure which exact scenarios would work better than our current logging system, but thought to allow that flexibility. However - if we want to keep the scope of this crate nice and focused then let's make the change - I'm happy either way. If so, should the CLI param be `--timing` ?

Reply:
Or `--time-profiling`. Probably you can add `telemetry` as default so we don't needed to type it (although a better default may be just `stdout`).

Reply:
that's a good point, stdout would be most useful for people I guess - I'll add that and include another CLI param to configure it

Reply:
also, just another thought regarding name - maybe `timing` or `timings` is too generic sounding - what about `profiling`?

Reply:
after some more consideration I have renamed to substrate-tracing, to align with the use of the tracing library for substrate, but also happy to switch to the above suggestion of @pepyakin if we want to be more focused/descriptive, my preference is slightly toward the current name, but not by much